Week-4/Exercise1.js

```diff
@@ -0,0 +1,26 @@
+const getNumber = (ms, printNumberCb) => {
+    return new Promise((resolve, reject)=>{
```

Comment:
Assignment says not to use default JS Promise & custom promise has to be created